src/main/java/nl/dtls/fairdatapoint/service/impl/FairMetaDataServiceImpl.java

```diff
@@ -86,6 +87,8 @@
 
     private final static Logger LOGGER
             = LogManager.getLogger(FairMetaDataServiceImpl.class);
+        
+    private static IRI fdpUri;
```

Comment:
Does this have to be static? Are there multiple instances of this class that share this field?

Reply:
Not really, I will remove the static keyword.